Point AAT at Preprod Vault section

Louisblack opened the pull request:
### Change description ###
* It should point to the preprod vault section.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

lgonczar commented:
send-letter also points to `test` from `aat`. This change will break it.

Louisblack commented:
Any chance of merging this in now that AAT and Test won't be peered from 5PM today?

alectronic0 commented on infrastructure/aat.tfvars:
```diff
@@ -0,0 +1,2 @@
+testing_support = "true"
```
this could actually be false now

Louisblack commented:
Updated PR to only point to preprod and updated title and description to match this. 
We are now using our vault secrets so do not need testing end points open.

Louisblack commented:
Looks like Jenkins and Git are having a bad time. I’ll have a butchers when I get in. Although when I looked yesterday, the master build was failing on Sonarqube so that might need to be looked at.
